NuRadioReco/modules/efieldToVoltageConverter.py

```diff
@@ -104,7 +104,9 @@ def run(self, evt, station, det):
         # for different cable delays
         times_min = []
         times_max = []
-        for iCh in det.get_channel_ids(sim_station_id):
+        if channel_ids is None:
+            channel_ids = det.get_channel_ids(sim_station_id)
```

Comment:
this could be:
```
channel_ids = channel_ids or det.get_channel_ids(sim_station_id)
```
Since I know the syntax I start liking it a lot. But not so many people do so it could make the code harder to read for those.